src/metabase/sync/analyze/classifiers/name.clj

```diff
@@ -129,4 +136,10 @@
                                     :googleanalytics :type/GoogleAnalyticsTable
                                     :druid           :type/EventTable
                                     nil)
-                                  :type/GenericTable))))
+                                  :type/GenericTable)]
+    (log/debug (format "Based on the name of %s, we're giving it entity type %s."
+                       (-> (table/->TableInstance)
+                           (merge table)
+                           sync-util/name-for-logging)
```

Comment:
Is there any reason we couldn't just do something like `(sync-util/name-for-logging (table/map->TableInstance table))` here? Instead of threading a bunch of things

Reply:
No, only that I forgot that map->Record exists. Fixed. Tnx.